features.h

```diff
@@ -12,6 +12,7 @@ struct qc71_features_struct {
 	bool fn_lock           : 1;
 	bool batt_charge_limit : 1;
 	bool fan_extras        : 1; /* duty cycle reduction, always on mode */
+	bool silent_mode       : 1; /* Slimbook turbo mode: increases fan rpm limit and tdp */
```

Comment:
To me, "silent mode" and "turbo mode" are the opposite ends of a spectrum, so which one is it?

Reply:
That's true, we will fix the comment